### ISSUE TITLE
Case sensitivity issue?

### ISSUE THREAD

**Issue by ThomBogers** [olistic/warriorjs#32]
When running warriorjs I get the following error:

```
thom@thom-laptop:~/Documents]~% warriorjs 
module.js:327
    throw err;
    ^

Error: Cannot find module '../../constants/directions'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/thom/.nvm/versions/node/v4.4.4/lib/node_modules/warriorjs/node_modules/warriorjs-engine/lib/abilities/actions/Attack.js:10:19)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/thom/.nvm/versions/node/v4.4.4/lib/node_modules/warriorjs/node_modules/warriorjs-engine/lib/LevelLoader.js:10:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
```

When I check the constants location I find the following:

```
[thom@thom-laptop:lib/constants]~% l                                  (v0.29.0)
total 108K
-rw-rw-r-- 1 thom thom 2,4K apr  8 06:18 Abilities.js
-rw-rw-r-- 1 thom thom  335 nov 15  2015 Actions.js
-rw-rw-r-- 1 thom thom  560 apr 10 20:12 Directions.js
-rw-rw-r-- 1 thom thom  308 apr 10 20:12 eventTypes.js
-rw-rw-r-- 1 thom thom  171 apr  2 00:57 logEvents.js
-rw-rw-r-- 1 thom thom  406 apr  8 06:18 RelativeDirections.js
-rw-rw-r-- 1 thom thom  335 nov 15  2015 Senses.js
-rw-rw-r-- 1 thom thom  998 apr  8 06:18 Units.js
-rw-rw-r-- 1 thom thom  303 nov 16  2015 UnitTypes.js
```

It seems warriorjs is looking for "directions.js" (lowercase) while the file name is "Directions.js"

When I symlink Directions.js to directions.js things seem to work fine

**Comment by olistic** [comment]
For some reason, the package `warriorjs-engine@0.10.0` in npm wasn't reflecting the exact output of the compilation using babel. I published a new version (v0.10.1), so running `npm uninstall -g warriorjs` and then `npm install -g warriorjs` should fix the issue.

**Comment by ThomBogers** [comment]
Works fine for me now, thanks!
